supervisord.conf

```diff
@@ -64,8 +64,8 @@ command =
         cfg:default.paths.data=/srv/grafana
         cfg:default.paths.plugins=/var/lib/grafana/plugins
         cfg:default.paths.logs=/srv/logs
-        cfg:default.log.mode=console
-        cfg:default.log.console.format=console
```

Comment:
Grafana should log to stderr with that configuration, and supervisord should capture that stream.
It doesn't work?

Reply:
Yes, you are right. So we have to rename /srv/logs/grafana.startup.log to /srv/logs/grafana.log